### PR TITLE
Add reconcile_interval environmental variable

### LINKED ISSUE #5: Add reconcileInterval to configuration
Add env-var to configure the `reconcileInterval` as `reconcile_interval` (Golang time.Duration)

This controls how often the idler tries to reconcile (apply idling if required). If this value is too frequent then we'll see additional load on the cluster.

Users will likely want to customize this value.

### PR DESCRIPTION
**Pull request by martindekov**
Adding reconcile_interval environmental variable to
configure the frequency

Signed-off-by: Martin Dekov (VMware) <mdekov@vmware.com>

Closes #5

### CODE REVIEW

File: faas-idler-dep.yml [hunk 1]
```diff
@@ -23,6 +23,8 @@ spec:
             value: "9090"
           - name: inactivity_duration
             value: "5m"
+          - name: reconcile_interval
+            value: "30"
```
Comment (alexellis) [inline faas-idler-dep.yml]: This should read `30s`.

File: main.go [hunk 2]
```diff
@@ -25,7 +25,11 @@ func main() {
 	flag.BoolVar(&dryRun, "dry-run", false, "use dry-run for scaling events")
 	flag.Parse()
 
-	reconcileInterval := time.Second * 30
+	frequency, freqErr := strconv.Atoi(os.Getenv("reconcile_interval"))
```
Comment (alexellis) [inline main.go]: The issue asks you to parse this as a `time.Duration`

File: main.go [hunk 3]
```diff
@@ -25,7 +25,11 @@ func main() {
...
+	if freqErr != nil {
+		log.Println(freqErr)
+	}
+	reconcileInterval := time.Second * time.Duration(frequency)
```
Comment (alexellis) [inline main.go]: This is now how to parse a duration, please Google it.

**Comment by alexellis** [comment]
Update the README too please - it currently says this value is hard-coded only.

Have you tested the change?

File: main.go (hunk 3 above)
Comment (martindekov) [reply main.go]: Ah ok

**Comment by martindekov** [comment]
No I will after I make the changes commented above.

File: main.go [hunk 4]
```diff
@@ -25,7 +25,15 @@ func main() {
 	flag.BoolVar(&dryRun, "dry-run", false, "use dry-run for scaling events")
 	flag.Parse()
 
-	reconcileInterval := time.Second * 30
+	frequency := os.Getenv("reconcile_interval")
```
Comment (alexellis) [inline main.go]: Please use `os.LookupEnv` instead.

Comment (alexellis) [reply main.go]: Here's a good example: https://github.com/openfaas/openfaas-cloud/blob/1e746b2d33682cebf110570d5006537e472de2f0/of-builder/main.go#L36

Can you turn this around before EOD?

**Pull request merged**